### DISCUSSION TITLE
Problems with yarpmanager cluster

### DISCUSSION THREAD

**Discussion by fvannu** [robotology/community#659] [Q&A] [answered]
Ciao!

We have a couple of problems with yarpmanager in the superbuild (latest yarp version 3.8.1) on ubuntu 20.04.6 LTS.
When we try to start a node from the cluster manager in yarpmanager, it says "yarprun succesfully executed on node", but actually nothing happens.
If we try to execute a command in the bar and hit "execute" (e.g. yarprun), the terminal log of yarpmanager says "bash: yarprun: command not found". The same commands launched from terminal work fine, we tried also to execute "env" on a terminal the result is this:

![](https://user-images.githubusercontent.com/23313360/276243280-aea877aa-2196-40bf-a647-a82cdef0e4cb.png)

while sending the "env" command from the yarpmanager box gives this result instead:

![](https://user-images.githubusercontent.com/23313360/276243741-55154c71-2d7f-40e1-aaa6-e3c5211e99d1.png)

In addition, we can't find from which cluster-config.xml file our cluster is built.

Do you have any idea no how to solve these issues?

**Comment by Nicogene** [comment, marked as answer]
Hi @fvannu,

since `ClusteWidget` uses `ssh` for running the `yarprun` and other commands, you have to set the superbuild environment variables **before** this statement in the `bashrc`: 

```bash
# If not running interactively, don't do anything
case $- in
    *i*) ;;
      *) return;;
esac

```

**Reply by fvannu** [reply]
Solved! thanks a lot for the help!
